packages/common/components/content/blocks/download-button.marko

```diff
@@ -0,0 +1,3 @@
+<cms-link-element class="btn btn-primary btn-lg" to=input.content path="fileSrc" target=input.target>
```

Comment:
Don't use the `btn` boostrap classes. Make a mixin and also apply to load more button so they use the same styles